CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Contributing
+
+Thank you for contributing to OpenShift Ansible. This document explains how the
+repository is organized, and how to submit contributions.
+
+## Introduction
+
+Before submitting code changes, get familiarized with these documents:
+
+- [Core Concepts](https://github.com/openshift/openshift-ansible/blob/master/docs/core_concepts_guide.adoc)
+- [Best Practices Guide](https://github.com/openshift/openshift-ansible/blob/master/docs/best_practices_guide.adoc)
+- [Style Guide](https://github.com/openshift/openshift-ansible/blob/master/docs/style_guide.adoc)
+
+## Repository structure
+
+### Scripts
+
+```
+.
+└── bin                 Contains the `bin/cluster` script, a wrapper around the
+                        Ansible playbooks that ensures proper configuration, and
+                        facilitates installing, updating, destroying and
+                        configuring OpenShift clusters.
+```
+
+### Documentation
+
+```
+.
+└── docs                Contains documentation for this repository.
+```
+
+### Tests
+
+```
+.
+└── test                Contains tests.
+```
+
+### Ansible
+
+```
+.
+├── inventory           Contains dynamic provisioning scripts, and examples of
+│                       Ansible inventories.
+├── library             Contains Python modules used by the playbooks.
+├── playbooks           Contains Ansible playbooks targeting multiple use cases.
+└── roles               Contains Ansible roles, units of shared behavior among
+                        playbooks.
+```
+
+#### Ansible plugins
+
+These are plugins used in playbooks and roles:
+
+```
+.
+├── ansible-profile
+├── callback_plugins
+├── filter_plugins
+└── lookup_plugins
+```
+
+<!-- TODO: document other directories
+
+```
+├── git
+└── utils
```

Comment:
Thanks @tbielawa! I'm understanding that `utils` is just called that instead of `atomic-openshift-installer` or `a-o-i` or `aoi` for some historic reasons?